src/Raven.Server/Storage/Schema/Updates/Server/62001/From62000.cs

```diff
@@ -0,0 +1,235 @@
+using System;
+using Raven.Server.Documents.Schemas;
+using Raven.Server.NotificationCenter.Notifications;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Sparrow.Server;
+using Voron;
+using Voron.Data.Tables;
+
+namespace Raven.Server.Storage.Schema.Updates.Server
+{
+    public class From62000 : ISchemaUpdate
+    {
+        public int From => 62_000;
+        public int To => 62_001;
```

Comment:
should this be 71_000 ?

Reply:
Since we have here schema upgrade, I think we should move this to 7.2 due to few reasons:
- I think that rolling back on a patch version should be doable
- in 7.2 we also have schema upgrade, so let's have them all in one version, will be easier to test and verify

thoughts @ppekrol @arekpalinski

Reply:
@Lwielit should be 71_000